### PR TITLE
Remove Real Svc Tests From Client Pipeline and Fix Test Results in Real Svc Test Pipeline

### PR DESCRIPTION
**Pull request by anthony-murphy**

### CODE REVIEW

File: tools/pipelines/build-client.yml [hunk 1]
```diff
@@ -111,4 +112,4 @@ extends:
           inputs:
             command: 'custom'
             workingDir: packages/test/end-to-end-tests
-            customCommand: 'run ci:test:realsvc' # This script ends with 'exit 0' so this task should never fail
+            customCommand: 'run ci:test:realsvc' # This script ends with 'exit 0' so this task should never fail
```
Comment (anthony-murphy) [inline tools/pipelines/build-client.yml:115]:
```suggestion
            customCommand: 'run ci:test:realsvc'
            
```

File: tools/pipelines/build-client.yml [hunk 2]
```diff
@@ -111,4 +112,5 @@ extends:
           inputs:
             command: 'custom'
             workingDir: packages/test/end-to-end-tests
-            customCommand: 'run ci:test:realsvc' # This script ends with 'exit 0' so this task should never fail
+            customCommand: 'run ci:test:realsvc'
+
```
Comment (anthony-murphy) [inline tools/pipelines/build-client.yml:116]:
```suggestion
```

File: tools/pipelines/test-real-service.yml [hunk 3]
```diff
@@ -70,6 +70,8 @@ jobs:
     # Run End-to-end Tests
     - task: Npm@1
       displayName: '[end-to-end tests] npm run ci:test:realsvc'
+      continueOnError: true
+      timeoutInMinutes: 0
```
Comment (anthony-murphy) [inline tools/pipelines/test-real-service.yml:74]:
```suggestion
```

File: tools/pipelines/build-client.yml [hunk 4]
```diff
@@ -99,16 +99,3 @@ extends:
           cp repo-package.json packages/package.json
           cp repo-package-lock.json packages/package-lock.json
 
-    additionalTests:
```
Comment (markfields) [inline tools/pipelines/build-client.yml:102]: I'd recommend removing this param from the template

File: tools/pipelines/test-real-service.yml [hunk 5]
```diff
@@ -22,7 +22,6 @@ parameters:
   type: boolean
   default: false
 
-trigger: none
```
Comment (markfields) [inline tools/pipelines/test-real-service.yml:25]: What does removing this do?

File: tools/pipelines/test-real-service.yml [hunk 6]
```diff
@@ -70,6 +69,7 @@ jobs:
     # Run End-to-end Tests
     - task: Npm@1
       displayName: '[end-to-end tests] npm run ci:test:realsvc'
+      continueOnError: true
```
Comment (markfields) [inline tools/pipelines/test-real-service.yml:72]: 😮

File: tools/pipelines/test-real-service.yml [hunk 7]
```diff
@@ -78,11 +78,12 @@ jobs:
       inputs:
         command: 'custom'
         workingDir: packages/test/end-to-end-tests
-        customCommand: 'run ci:test:realsvc' # This script ends with 'exit 0' so this task should never fail
+        customCommand: 'run ci:test:realsvc'
```
Comment (markfields) [inline tools/pipelines/test-real-service.yml:81]: Is this script updated in another PR?

**Review by markfields — CHANGES_REQUESTED** [review]
🎂 Take a look at my comments but overall LTGM!

File: tools/pipelines/test-real-service.yml (hunk 5 above)
Comment (anthony-murphy) [reply tools/pipelines/test-real-service.yml:25]: it will run as a CI build, so after every change

**Review by markfields — APPROVED** [review]

File: tools/pipelines/test-real-service.yml (hunk 7 above)
Comment (anthony-murphy) [reply tools/pipelines/test-real-service.yml:81]: i removed exit 0 already, build is broken

File: tools/pipelines/build-client.yml (hunk 4 above)
Comment (anthony-murphy) [reply tools/pipelines/build-client.yml:102]: i can do that is a follow up, as this branch is annoying as it's not in my fork, and i block pushing to upstream from my local client, as i'm admin and can break stuff on accident

**Pull request merged**